Cargo.toml

```diff
@@ -8,6 +8,8 @@ members = [
   "crates/astria-sequencer",
   "crates/astria-sequencer-client",
   "crates/astria-sequencer-relayer",
+  "crates/astria-sequencer-validation",
+  "crates/astria-sequencer-types",
```

Comment:
Should the types here have been added in a different PR seems like might warrant a seperate small quick PR?

Reply:
yeah, it doesn't seem to have any ill effect right now but will add